nginx/html/portal-ui-maintenance/index.html

```diff
@@ -1 +1 @@
-<!doctype html><html><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no"><title>Maintenance | HuBMAP</title><link rel="icon" href="/favicon.ico"></head><body><div id="react-content"></div><script src="/maintenance.bundle.js"></script></body></html>
\ No newline at end of file
+<!doctype html><html><head><meta charset="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no"/><title>Maintenance | HuBMAP</title><script>const sentryEnv = '{{ sentry_env }}';</script><link rel="icon" href="favicon.ico"><script defer="defer" src="maintenance.bundle.js"></script></head><body><div id="react-content"></div></body></html>
```

Comment:
What is this `sentry_env`? Do I need to do anything specially?

Reply:
`sentry_env` is a variable for Portal tracking. There's nothing that needs to be done for that. 


This PR is an updated hardcoded message for one-time use. We'll address making the messaging configurable in a later update.